docs/content/ingestion/faq.md

```diff
@@ -2,6 +2,14 @@
 layout: doc_page
 ---
 
+## My Data isn't being loaded
+
+If you are trying to stream in historical (not current time) data into Druid and you are using the [serverTime](../ingestion/realtime-ingestion.html) rejection policy in your ingestion spec (the default rejection policy), Druid will not ingest this data as it is outside of the acceptable window period. You can verify this is what is happening by looking at the logs of your real-time process for log lines containing "ingest/events/*". These metrics will indicate the events ingested, rejected, etc. We recommend using batch ingestion methods for historical data in production.
+
+If you are doing a POC, you can use the [messageTime](../ingestion/realtime-ingestion.html) rejection policy, but please be aware of the handoff caveats. This rejection policy is not recommended in production. You can also use the [none](../ingestion/realtime-ingestion.html) rejection policy to load all incoming events, but hand-off will never occur. The only recommended rejection policy for production is serverTime. 
```

Comment:
I think it's a little confusing which advice applies to realtime and which applies to batch. Maybe that could be fixed by one of the following,

- Saying "realtime POC" or saying "realtime ingestion" / "batch ingestion" at the start of each of these paragraphs, to make it clear that this is about realtime and not batch

- Splitting this question into two questions, one for the rt version and batch version of "why isn't my data being loaded"